### DISCUSSION TITLE
New website design!

### DISCUSSION THREAD

**Discussion by Felienne** [hedyorg/hedy#3913] [Announcements]
Since this week, our website has a new design!

![image](https://user-images.githubusercontent.com/1003685/210571031-0a113771-829e-4a96-a2b1-9fce8edf47a1.png)

The code page is refreshed too:

![image](https://user-images.githubusercontent.com/1003685/210571068-8ab3e9f9-c2ea-41fc-b7c5-fe66547cfe77.png)

**Comment by Felienne** [comment]
We hope the new site communicates more clearly what Hedy is!

**Comment by redelaar** [comment]
Looks great!
